tests/test_swift.py

```diff
@@ -104,17 +108,23 @@ def test_destroy_app(self):
             for channel in info.channel:
                 self.assertNotIn(name, self.swift._subscribers[channel])
 
-    def test_update_frames(self):
+    def test_update_frames_inclusive(self):
```

Comment:
I think the meaning of "inclusive" and "exclusive" here is not so obvious.
Writing a docstring for these tests would help the readers to understand.

Reply:
Okay! I will add some descriptions.